app/controllers/admin/mentors_controller.rb

```diff
@@ -1,20 +1,20 @@
 class Admin::MentorsController < ApplicationController
   def index
-    authorize Mentor
+    authorize User, :mentors?
 
-    render json: Mentor.includes(:careers), each_serializer: Admin::MentorSerializer
+    render json: User.where(mentor: true).includes(:careers, :traits), each_serializer: Admin::MentorSerializer
   end
 
   def show
-    mentor = Mentor.find(params[:id])
+    mentor = User.find(params[:id])
     authorize mentor
 
     render json: mentor, serializer: Admin::MentorSerializer
   end
 
   def create
-    authorize Mentor
-    mentor = Mentor.new(mentor_params.merge(user_attributes: user_params))
+    authorize User, :admin?
```

Comment:
Why is this action the exception, using `#admin?`?

Reply:
Because only an admin can create a new User, here

Reply:
only an admin can do anything, here. That's why I feel its a bit weird. I think we should just expand the `create?` policy to contemplate this action as well.

Reply:
```
  def create?
    !@user || @user&admin?
  end
```

?

Reply:
`!@user || @user.admin?`, I guess.

Reply:
Right, no need for safe access here 👌